monitor/services/src/main/java/org/datacleaner/monitor/configuration/ComponentsFactory.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.monitor.configuration;
+
+import org.datacleaner.monitor.server.components.ComponentHandler;
+
+/**
+ * Class ComponentsFactory
+ *
+ * @author k.houzvicka
+ * @since 18.8.15
+ */
+public class ComponentsFactory {
```

Comment:
And this one :)
Maybe it should actually be renamed to ComponentHandlerFactory, as this is what it produces.